main.js

```diff
@@ -36,6 +40,113 @@ async function parseJiraCsv(filename) {
   handle.close();
 }
 
+async function createIssue(apiToken, name, description, stage) {
+  const createBlob = JSON.stringify({
+    "fields": {
+      "project": {
+        "key": PROJECTKEY,
+      },
+      "parent": {
+        "key": STAGE_EPICS[stage]
+      },
+      "summary": name,
+      "issuetype": {
+        "id": ISSUETYPE_STORY,
+      },
+      "description": description,
+      "components": [{"name": COMPONENT}],
+    }
+  });
+
+  const options = {
```

Comment:
I think this looks good! 

Using `fetch` could simplify the code and remove the need to import `https` and `node-fetch`.

Something like this:
```js
const response = await fetch("https://mozilla-hub.atlassian.net/rest/api/2/issue/", {
  method: "POST",
  headers: {
    "Authorization": "Basic " + new Buffer.from(`${USER}@mozilla.com:${apiToken}`).toString('base64'),
    "Content-Type": "application/json",
    "Content-Length":`${createBlob.length}`
  },
  body: createBlob
});

if (!response.ok) {
  console.error(`error: could not create ${name}: `, error);
} else {
  const { key, errors } = await response.json();
  if (key !== undefined) {
    console.log(`Created issue for ${name} with key: ${key}`);
  } else {
    console.log(`Error creating issue for ${name}: ${JSON.stringify(errors)}`);
  }
}
```